java-package/openbci_gui_helpers/src/main/java/openbci_gui_helpers/GUIHelper.java

```diff
@@ -1,128 +1,128 @@
 package openbci_gui_helpers;
 
-import java.io.BufferedReader;
 import java.io.File;
 import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.lang.reflect.Type;
-import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.nio.file.Path;
-import java.util.Map;
-import java.util.stream.Collectors;
+import java.util.ArrayList;
 
 import org.apache.commons.lang3.SystemUtils;
 
 import com.google.gson.Gson;
-import com.google.gson.reflect.TypeToken;
 import com.sun.jna.Library;
 import com.sun.jna.Native;
 
-public class GUIHelper
-{
-    private interface DllInterface extends Library
-    {
-        int scan_for_ganglions (String serial_port, int timeout_sec, byte[] output, int[] output_len);
+public class GUIHelper {
+    public class GanglionDevice {
```

Comment:
I see best practice here extracting this to it's own class. Makes sense since you added a third value.